### PR TITLE
Expands LogRecord to allow "anyValue" in body parameter.

### PR DESCRIPTION
**Pull request by bryce-b**

### CODE REVIEW

**Comment by vvydier** [comment]
That looks like the right way to do it!  I created the spec-change branch after I modified the ActivityContextManager code and test case in my main, so it also shows up here. Do you think it would be ok to merge this or do we need to remove the ActivityContextManager updates from this PR?

**Review by vvydier — APPROVED** [review]
Looks good to me!

File: Sources/OpenTelemetryApi/Context/ActivityContextManager.swift [hunk 1]
```diff
@@ -68,14 +68,12 @@ class ActivityContextManager: ContextManager {
...
-           currentValue === value
-        {
-         contextMap[activityIdent]?[key.rawValue] = nil
-          if contextMap[activityIdent]?.isEmpty ?? false {
-            contextMap[activityIdent] = nil
+        
+        for (activityKey, activity) in contextMap where value === activity[key.rawValue] {
+            contextMap[activityKey]?[key.rawValue] = nil
+          if contextMap[activityKey]?.isEmpty ?? false {
+            contextMap[activityKey] = nil
```
Comment (nachoBonafonte) [inline Sources/OpenTelemetryApi/Context/ActivityContextManager.swift:76]: These changes don't look related to this PR

File: Tests/OpenTelemetryApiTests/Context/ActivityContextManagerTests.swift [hunk 2]
```diff
@@ -308,6 +308,45 @@ class ActivityContextManagerTests: XCTestCase {
         XCTAssert(OpenTelemetry.instance.contextProvider.activeSpan === nil)
     }
 
+
```
Comment (nachoBonafonte) [inline Tests/OpenTelemetryApiTests/Context/ActivityContextManagerTests.swift:311]: this also looks like code not related to these changes

**Review by nachoBonafonte — CHANGES_REQUESTED** [review]
LGTM, but there are some changes that I think are part of another PR from @vvydier

**Review by nachoBonafonte — APPROVED** [review]

**Pull request merged**